iOS_sdk_source/Demo/VeepooBleSDKDemo/VeepooBleSDKDemo/VeepooSDKViewControllers/VPTestOxygenController.swift

```diff
@@ -16,25 +16,55 @@ class VPTestOxygenController: UIViewController {
     @IBOutlet weak var testOxygenBtn: UIButton!
     @IBOutlet weak var testRateBtn: UIButton!
     
+    @IBOutlet weak var oxygenDateLabel: UILabel!
+    
+    // Graph to display oxygen data
+    var oxygenCurview = VPOxygenCurveView()
+    
+    // Index to the current day
+    var oxygenDayIndex = 0
+    
+    var y: CGFloat = 0.0
+    var width: CGFloat = 0.0
```

Comment:
Shouldn't we choose names with a little of more detail/information, when they are going to be used as global variables? The variables "y" and "width" are too much generic
And "oxygenCurview" shouldn't be camel case all the way, like "oxygenCurView"?